lib/cmd/common/apps.js

```diff
@@ -68,15 +74,15 @@ function oldApps(args, cb){
 
   var action = args[0];
   if (action === 'read') {
-    return cb("The 'fhc apps read' command is deprecated. Please use 'fhc read' command instead.");
+    return cb(errorMessageString(action));
   }else if (action === 'create'){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either merge this branch with the identical one on line "76" or change one of the implementations. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS1871)